datalake/backend/endpoints/utils.py

```diff
@@ -13,3 +17,13 @@ def success_response(code, body):
     """Return success response content as string."""
 
     return {"statusCode": code, "body": body}
+
+
+class ResourceName(Enum):
+    DATASETS_TABLE_NAME = f"{ENV}-datasets"
+    DATASETS_TITLE_INDEX_NAME = "datasets_title"
+    DATASETS_OWNING_GROUP_INDEX_NAME = "datasets_owning_group"
```

Comment:
As discussed, these two should probably be separate constants.

Reply:
resolved this.